### ISSUE TITLE
Fedora asking for remote desktop interaction

### ISSUE THREAD

**Issue by kema-dev** [hluk/CopyQ#2565]
**Describe the bug**

Fedora asking for Remote Desktop sharing on clip selection when VSCode is openned.

**To Reproduce**

1. Open copyq with a keyboard shortcut
2. Select a clip

**Expected behavior**

Text copied to clipboard, but no Remote Desktop pop-up

**Screenshots**

![image](https://github.com/hluk/CopyQ/assets/54537427/c44a42ae-0252-436f-b577-58342780a548)

**Version, OS and Environment**

copyq

```text
CopyQ Clipboard Manager 7.1.0
Qt: 5.15.10
KNotifications: 5.109.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Fedora Linux 39 (Workstation Edition)
```

os

```
Resolution: 1920x1080 
DE: GNOME 45.2 
WM: Mutter 
WM Theme: Adwaita 
Theme: Adwaita [GTK2/3] 
Icons: Adwaita [GTK2/3] 
```

**Additional context**

I am using Wayland, might be another electron-related bug. Maybe I'll dig a bit deeper into this bug in the weekend.

**Comment by roman7722** [comment]
I have the same problem. Did you manage to solve it?

copyq 8.0.0-1
Manjaro Linux
Wayland
Linux 5.15.160-1-MANJARO

**Comment by kema-dev** [comment]
I could not figure out how to solve this issue, thus stopped using CopyQ and switched to a custom bash / fzf / Alacritty clipboard manager

**Comment by roman7722** [comment]
> Я не мог понять, как решить эту проблему, поэтому прекратил использовать CopyQ и переключился на собственный менеджер буфера обмена bash/fzf/Alacritty.

Thanks for the answer, colleague!

**Comment by kema-dev** [comment]
You might want to try upgrading GNOME (if you're using GNOME, though), v46 fixed some bugs for me :shrug:

**Comment by roman7722** [comment]
> You might want to try upgrading GNOME (if you're using GNOME, though), v46 fixed some bugs for me 🤷

Yes, I use Gnome v.46. Just after the update this crash happened.

**Comment by hluk** [comment]
By default, after activating an item in CopyQ, it sends **Shift+Insert** shortcut to the target window to paste clipboard. Could that cause the issue.

BTW, new versions of GNOME and distros often use Wayland session by default. This may also cause some problems, see: https://copyq.readthedocs.io/en/latest/known-issues.html#known-issue-wayland

**Comment by rgarcia89** [comment]
I am facing the same issue. Would be more than interested in a solution for it

**Comment by rgarcia89** [comment]
I just added https://github.com/hluk/copyq-commands/blob/master/Scripts/wayland-support.ini as command to copyq. Now things are working fine again. Just download the file, open the copyq user interface (not menu), press F6, Load Commands, load the wayland-support.ini. Done it is

**Comment by tomergam** [comment]
There is any solution for copyq with wayland fedora 41?
Thanks

**Comment by yuriw** [comment]
Me too on F41
Any clues how to fix it?

**Comment by yuriw** [comment]
I can't believe this issue is here since 2023!!!

Can someone comment with any thoughts or recommendations??
(except switch to Windows 11 :) )

**Comment by hluk** [comment]
> I can't believe this issue is here since 2023!!!

I have never encountered the issue (I use Fedora 41!), and I don't understand why it would happen. It looks like a GNOME bug.

As mentioned above, if you use Wayland on GNOME, you need to either switch the app to XWayland/X11 mode or use the [Wayland Support](https://github.com/hluk/copyq-commands/tree/master/Scripts#wayland-support) command. See: https://copyq.readthedocs.io/en/latest/known-issues.html#known-issue-wayland

BTW, in the next release, there are more fixes coming for Wayland and some built-in workarounds for GNOME.

**Comment by heyeddi** [comment]
> I just added https://github.com/hluk/copyq-commands/blob/master/Scripts/wayland-support.ini as command to copyq. Now things are working fine again. Just download the file, open the copyq user interface (not menu), press F6, Load Commands, load the wayland-support.ini. Done it is

This is the way

**Comment by yuriw** [comment]
It seems to work I get an error on pressing Enter

![Image](https://github.com/user-attachments/assets/d135fff4-ca81-4e7f-b486-076e09614176)

@heyeddi thx!!!

**Comment by hluk** [comment]
@yuriw That is related to [ydotool](https://github.com/ReimuNotMoe/ydotool), which may not be trivial to set up (I don't use it anymore - I can live without the "paste from CopyQ" functionality).

**Comment by yuriw** [comment]
> [@yuriw](https://github.com/yuriw) That is related to [ydotool](https://github.com/ReimuNotMoe/ydotool), which may not be trivial to set up (I don't use it anymore - I can live without the "paste from CopyQ" functionality).

I think I don't get text paste on Enter anymore
Do I need to disable it somehow, or just ignore?

**Comment by hluk** [comment]
> I think I don't get text paste on Enter anymore
> Do I need to disable it somehow, or just ignore?

You can disable it via Preferences - History tab - option "Paste to current window".

**Comment by yuriw** [comment]
> > I think I don't get text paste on Enter anymore
> > Do I need to disable it somehow, or just ignore?
> 
> You can disable it via Preferences - History tab - option "Paste to current window".

Done, thx

One issue I still see after adding https://github.com/hluk/copyq-commands/blob/master/Scripts/wayland-support.ini my shortcut keys won't work at all

**Comment by hluk** [comment]
> One issue I still see after adding https://github.com/hluk/copyq-commands/blob/master/Scripts/wayland-support.ini my shortcut keys won't work at all

Wayland does not support managing global shortcuts in apps. You need to set them in the desktop environment instead.

**Comment by kema-dev** [comment]
Looks like this is no longer an issue then, closing as completed

**Comment by tsMukesh51** [comment]
I got issue, I'm getting this from a month I guess. I'm on Fedora 42. And I added `env QT_QPA_PLATFORM=xcb`. I observed is if I allow the remote interaction it will paste the text into the textbox, else simply copies. How can I  provide logs, please share a guide link.
